vignettes/nactdt.Rmd

```diff
@@ -0,0 +1,220 @@
+)
+```
+
+## Single Source{#input}
+
+To derive the New Anti-Cancer Therapy start date with data from a single source, the function `admiral::derive_vars_merged()` can be used. If numeric date is required,  `admiral::convert_dtc_to_dt` , or if time part is needed, `admiral::convert_dtc_to_dtm()` can be used as part of an expression for the `new_vars` argument.
+
+```{r message=FALSE}
+
+adsl <- derive_vars_merged(
+  admiral_adsl,
+  dataset_add = cm,
+  by_vars = exprs(STUDYID, USUBJID),
+  order = exprs(NACTDT),
+  mode = "first",
+  new_vars = exprs(NACTDT = convert_dtc_to_dt(CMSTDTC)),
+  filter_add = CMSCAT == "CHEMOTHERAPY" & CMCAT == "ON TREATMENT"
+)
+```
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  adsl,
+  display_vars = exprs(USUBJID, NACTDT),
+  filter = !is.na(NACTDT)
+)
+```
+
+## Multiple Sources - Prerequisite Steps {#multiinput}
+
+To derive the New Anti-Cancer Therapy start date with data from multiple sources, the function `admiral::derive_var_extreme_dt()` can be used, or if time part is needed, `admiral::derive_var_extreme_dtm()` can be used.
+ 
+ 
+### Prerequisite 
+
+Prior to calling the function,an input `date_source` object must be created for each source dataset using `admiral::date_source()`. Within this function, any company-specific subsetting can be passed to the `filter` argument. if numeric version of the dates will be needed,`admiral::convert_dtc_to_dt`, or if time part is needed, `admiral::convert_dtc_to_dtm` can be used as part of an expression for the `new_vars` argument. 
+
+```{r message=FALSE}
+cm_date <- date_source(
+  dataset_name = "cm",
+  filter = CMSCAT == "CHEMOTHERAPY" & CMCAT == "ON TREATMENT" & !is.na(CMSTDTC),
+  date = convert_dtc_to_dt(CMSTDTC)
+)
+
+pr_date <- date_source(
+  dataset_name = "pr",
+  filter = PRCAT == "CANCER RELATED" & PRSCAT == "ON TREATMENT" & !is.na(PRSTDTC),
+  date = convert_dtc_to_dt(PRSTDTC)
+)
+```
+
+## Multiple Sources - Derive New Anti-Cancer Start Date and Merge with `ADSL` {#nactdt}
+
+After completion of the prerequisite step, the new anti-cancer date can be derived while simultaneously adding it to `ADSL`.
+
+```{r message=FALSE}
+adsl <- admiral_adsl %>%
+  derive_var_extreme_dt(
+    new_var = NACTDT,
+    cm_date, pr_date,
+    source_datasets = list(
+      cm = cm,
+      pr = pr
+    ),
+    mode = "first"
+  )
+```
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  adsl,
+  display_vars = exprs(USUBJID, NACTDT),
+  filter = !is.na(NACTDT)
+)
+```
+## Derive Date Parameter {#par}
+
+For deriving parameter for new anti-cancer date from a single source, `admiral::derive_param_extreme_event()` may be used, please consult the `{admiralonco}` [Creating ADRS](adrs.html) for guidance on creating parameters and on the usage of `NACTDT` variable with examples. Additionally, subsection *Creating Your Own Time-to-Event Source Objects* in the `{admiralonco}` [Creating ADTTE](adtte.html) vignette may be referenced for case usage.
```

Comment:
we have now removed admiral::derive_param_extreme_event from adrs vignette as well? @bundfussr is it will be correct to mention it now "please consult the `{admiralonco}` [Creating ADRS](adrs.html) for guidance on creating parameters and on the usage of `NACTDT` variable with examples. " I think we should mentioned derive_param_extreme_record() instead of param_extreme_event() function in the text?

Reply:
Yes, `derive_param_extreme_event()` is deprecated in favor of `derive_extreme_records()` in admiral 0.11.0.